# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1548356158 0
#      Thu Jan 24 18:55:58 2019 +0000
# Branch stable
# Node ID 3945d8dd56a1ea60d998148684f7ae3c29880858
# Parent  725f3f01f487c7fa5d1e3bec33e31f53c356285f
* HACKING.md: Explain release candidate version numbers.

diff --git a/etc/HACKING.md b/etc/HACKING.md
--- a/etc/HACKING.md
+++ b/etc/HACKING.md
@@ -301,34 +301,37 @@ Release Numbering
 -----------------
 
 Since version 5, Octave uses the following rules for release numbering:
 
   Version Dev Phase       When
 
   5.0.0   (experimental)  active development of Octave 5 on default branch
   5.0.1   (pre-release)   stabilization period of Octave 5 on stable branch
+  5.0.90  (pre-release)   first release candidate for 5.1.0
   6.0.0   (experimental)  active development of Octave 6 on default branch
   5.1.0   (release)       first release of Octave 5 from stable branch
   5.1.1   (pre-release)   bug fixing on stable branch after 5.1.0 release
   5.2.0   (release)       second release of Octave 5 from stable branch
   5.2.1   (pre-release)   bug fixing on stable branch after 5.2.0 release
   ...
 
 To summarize, the first release of Octave 5 will be Octave 5.1.0 while
 development snapshots will be Octave 5.0.0 and snapshots from the
 release branch Octave 5.n.1.
 
 With this numbering scheme:
 
   * Any version X.0.0 means "this is an experimental development
     version".
 
-  * Any version X.Y.1 means, "this is a pre-release version meant
-    for bug fixing and testing".
+  * Any version X.Y.Z with Z > 0 means, "this is a pre-release version
+    meant for bug fixing and testing".  In practice, Z will be either 1
+    (stabilization period after and before making release candidates) or
+    90, 91, etc. (release candidate snapshots leading up to release).
 
   * Any version X.Y.0 with Y != 0 means "this is a released version".
 
 Shared Library Versioning
 -------------------------
 
 Version numbers for the liboctave, liboctinterp, and liboctgui shared
 libraries are set in the module.mk files in the top-level directory for
